bot/cogs/bot.py

```diff
@@ -60,6 +62,18 @@ def __init__(self, bot: AutoShardedBot):
 
         await ctx.invoke(self.info)
 
+    async def on_message(self, msg: Message):
+        """
+        Auto codeblock python code
```

Comment:
Is this docstring necessary? In future it may not just be used for parsing Python code.

Reply:
Yes, one could, of course, delete it but just for current usage. Depends on you del, not del?

Reply:
If you look at the other files where event listeners are defined they don't have docstrings, to stay compliant, this should be removed.